docs/source/support/faq.rst

```diff
@@ -112,7 +112,7 @@ sets of keys separate.
 Q: How can I get help integrating nucypher into my application?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-See :ref:`application-development`. To chat directly with our team, please join our `Discord <http://discord.nucypher.com>`_.
+See :doc:`/getting_started`. To chat directly with our team, please join our `Discord <http://discord.nucypher.com>`_.
```

Comment:
any reason you prefer `doc` to `ref`? Just curious.

Reply:
No presence, I actually didn't notice this change.